go.mod

```diff
@@ -4,7 +4,7 @@ go 1.17
 
 require (
 	github.com/google/go-cmp v0.5.8
-	github.com/hashicorp/terraform-plugin-framework v0.9.0
+	github.com/hashicorp/terraform-plugin-framework v0.9.1-0.20220627174514-5a338a7dd906
```

Comment:
Just putting a note here so we either rebase this after the dependabot PR for v0.10.0 or update this PR for v0.10.0 when its released.